xl2times/datatypes.py

```diff
@@ -266,6 +267,20 @@ def model_years(self) -> set[int]:
         """
         return self.past_years | set(self.time_periods["m"].values)
 
+    @cached_property
+    def veda_cgs(self) -> dict[tuple[str, str, str], str]:
+        """veda_cgs is a dictionary mapping commodities to their Veda commodity groups."""
```

Comment:
```suggestion
        """A dictionary mapping commodities to their Veda commodity groups."""
```

Just double checking: we don't expect this value to change when other properties/fields of the dataclass changes? Maybe we should add a TODO to check when this cached property is invalidated and have a way to clear the cache / recompute this. I'm wondering if that will be useful once we allow people to work with `TimesModel` objects directly in e.g. Jupyter Notebooks?

Reply:
Sounds like a good idea!